lib/player.js

```diff
@@ -5445,7 +5446,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         return new shaka.text.UITextDisplayer(
             this.video_, this.videoContainer_);
       } else {
-        return new shaka.text.SimpleTextDisplayer(this.video_);
+        if ('addTextTrack' in this.video_) {
```

Comment:
Can we use `HTMLMediaElement.prototype.addTextTrack`  instead?

Reply:
Done - thanks

Reply:
Doh, i just saw that this requires a lint disable, `// eslint-disable-next-line no-restricted-syntax` ?

Reply:
Yes, It's necessary

Reply:
ack thanks again